server/server.go

```diff
@@ -108,6 +109,7 @@ func (s *Server) Shutdown() {
 	s.shutdownOnce.Do(func() {})
 }
 
+// Here we ensure that certain error types are handled specifically.
```

Comment:
I'd rather drop that comment. We have that function everywhere. Also godoc starts with function name.

Reply:
Yeah, I know. I am somewhat driven to remove those nitpicks by golint/govet my IDE shows me all the time, so that I don't miss those that make sense. I also like to read these natural language comments in order to get to know a package better. Even if they may be common sense for those more familiar with it.

Reply:
So please make it a good godoc at least.

Reply:
done